vars/fcosKola.groovy

```diff
@@ -34,6 +35,12 @@ def call(params = [:]) {
         }
         def parallel = params.get('parallel', 8);
         def extraArgs = params.get('extraArgs', "");
+        def addExtTests = params.get('addExtTests', [])
+
+        for (path in addExtTests) {
+            args += "--exttest ${env.WORKSPACE}/${path}"
```

Comment:
I think we need a leading space here to make sure that they're separate args. I.e.

```suggestion
            args += " --exttest ${env.WORKSPACE}/${path}"
```